project/Dependencies.scala

```diff
@@ -103,6 +103,9 @@ object Dependencies {
   val scalaTest            = "org.scalatest"        %% "scalatest"                 % "3.2.2"         // Scala 3 compatible
   val testcontainers       = "org.testcontainers"    % "testcontainers"            % "1.16.3"
 
+  // found/added by the plugin but deleted anyway
+  val commonsLang3 = "org.apache.commons" % "commons-lang3" % "3.12.0"
```

Comment:
what's the deal with that? I don't understand the comment

Reply:
Long story short: a plugin detected bunch of undeclared deps for webapi project, so there were added and removed after a while, since didn't change anything. However I needed to declare one of these now, because used in copied code. The comment is just to distinguish deps.